docs/datamodel/scalars/numeric.rst

```diff
@@ -156,17 +166,21 @@ from :eql:type:`str` and :eql:type:`json`.
         db> SELECT 1.23n IS decimal;
         {true}
 
-        db> SELECT 1e+100n IS decimal;
+        db> SELECT 1.0e+100n IS decimal;
         {true}
 
-    Note that an integer literal (without a dot or exponent) followed by 'n'
-    produces a :eql:type:`bigint`:
+    Note that an integer literal (without a dot or exponent) followed
+    by 'n' produces a :eql:type:`bigint` as well as a literal with a
+    positive exponent is and no dot in the mantissa:
```

Comment:
This sentence is too long. I needed to read it three times to make sense of it.

Reply:
I've split it and reworded a bit.